data/arena.txt

```diff
@@ -4443,4 +4443,253 @@ fleet "Emerald Sword "
+	tribute 1
+		fleet "Ojde "
+		threshold 0
+fleet "Ojde "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Ojde"
+planet "Kvelqde-Khora "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Kvelqde-Khora "
+		threshold 0
+planet "Kvelqde-Khora  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Kvelqde-Khora "
+		threshold 0
+fleet "Kvelqde-Khora "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Kvelqde-Khora"
+planet "Stolsaqra "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Stolsaqra "
+		threshold 0
+planet "Stolsaqra  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Stolsaqra "
+		threshold 0
+fleet "Stolsaqra "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Stolsaqra"
+planet "Vujlet "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Vujlet "
+		threshold 0
+planet "Vujlet  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Vujlet "
+		threshold 0
+fleet "Vujlet "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Vujlet"
+planet "Veusade  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Veusade "
+		threshold 0
+planet "Veusade  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Veusade "
+		threshold 0
+fleet "Veusade "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Veusade"
+planet "Seiitej "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Seiitej "
+		threshold 0
+planet "Seiitej  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Seiitej "
+		threshold 0
+fleet "Seiitej "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Seiitej"
+planet "Aaulqra "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Aaulqra "
+		threshold 0
+planet "Aaulqra  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Aaulqra "
+		threshold 0
+fleet "Aaulqra "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Aaulqra"
+planet "Kaskhorade "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Kaskhorade "
+		threshold 0
+planet "Kaskhorade  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Kaskhorade "
+		threshold 0
+fleet "Kaskhorade "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Kaskhorade"
+planet "Nnaatej "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Nnaatej "
+		threshold 0
+planet "Nnaatej  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Nnaatej "
+		threshold 0
+fleet "Nnaatej "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Nnaatej"
+planet "Valkkyiara-Khorasaqra "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Valkkyiara-Khorasaqra "
+		threshold 0
+planet "Valkkyiara-Khorasaqra  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Valkkyiara-Khorasaqra "
+		threshold 0
+fleet "Valkkyiara-Khorasaqra "
+	government "Arena"
+	personality "heroic"
+	variant
+		"Valkkyiara-Khorasaqra"
+planet "Kijra-Khora  "
+	bribe 0
+	government "Arena"
+	tribute 1
+		fleet "Kijra-Khora "
+		threshold 0
+fleet "Kijra-Khora "
+	government "Arena"
+	personality heroic
+	variant
+		"Kijra-Khora"
```

Comment:
Neither the V-K or Kijra-Khora are included in the initial PR (because they're not used in-game). May be best to keep these lines, and others referencing those ships, out until later.